web-ui/github.ml

```diff
@@ -36,6 +36,26 @@ let github_branch_url ~owner ~name ref =
 let github_pr_url ~owner ~name id =
   Printf.sprintf "https://github.com/%s/%s/pull/%s" owner name id
 
+let css = {|
+  .statuses {
+    list-style: none;
+  }
+  .statuses > li:before {
+    display: inline-block;
+    width: 1em;
+    margin-right: 0.5em;
+    margin-left: -1.5em;
+    text-align: center;
+    line-height: 1.1em;
+  }
+  .statuses > li.not-started:before { content: "●"; color:grey; }
+  .statuses > li.aborted:before { content: "●"; color:red; }
```

Comment:
```suggestion
  .statuses > li.aborted:before { content: "A"; color:red; }
```
I still prefer 'A' here, as is being done in Citty. A red dot doesn't look like 'aborted' to me, and this symbol is already being shared.